R/modules.R

```diff
@@ -315,7 +313,12 @@ modules <- function(..., label = "root") {
 
 #' @rdname teal_modules
 #' @export
-format.teal_module <- function(x, indent = 0, ...) {
+format.teal_module <- function(x, ...) {
```

Comment:
removed `indent` from the `module()` documentation. It is completely unnecessary to display `indent` in the public docs.

Reply:
cc1f7bec13c0b80eb05f809cc914ad06297b82e0